dumpling: add URI formats (#15165)
pingcap/docs: ticdc/ticdc-sink-to-cloud-storage.md

> **Tip:**
>

> > The URI parameters of Amazon S3, GCS, and Azure Blob Storage in TiCDC are the same as their URI parameters in BR. For details, see [URI Formats of External Storage Services](/external-storage-uri.md).

Author: ```suggestion
> For more information about the URI parameters of Amazon S3, GCS, and Azure Blob Storage in TiCDC, see [URI Formats of External Storage Services](/external-storage-uri.md).
```
